pincer/objects/app/interactions.py

```diff
@@ -145,33 +145,36 @@ def __post_init__(self):
             return
 
         for option in self.data.options:
-            if option.type is AppCommandOptionType.STRING:
+            if option.type == AppCommandOptionType.STRING:
                 option.value = str(option.value)
-            elif option.type is AppCommandOptionType.INTEGER:
+            elif option.type == AppCommandOptionType.INTEGER:
                 option.value = int(option.value)
-            elif option.type is AppCommandOptionType.BOOLEAN:
-                option.value = bool(option.value)
-            elif option.type is AppCommandOptionType.NUMBER:
+            elif option.type == AppCommandOptionType.BOOLEAN:
+                option.value = option.value == "True"
```

Comment:
I'm not sure of this change

Reply:
the problem is that discord send "True" or "False" as str and i don't know how to make it boolean with the pep8 style